main.py

```diff
@@ -50,7 +50,7 @@ def func_tokens():
 
 github.enable_console_debug_logging()
 g = github.Github(j[1])
-client = discord.Client()
+client = discord.AutoShardedClient(shards=20)
```

Comment:
IIRC, you don't need to put the amount of shards, and also it's ```AutoShardedBot```

Reply:
Gotta fix that then, thanks